roles/srcds/templates/server.cfg.j2

```diff
@@ -74,7 +74,7 @@ mp_forcecamera 1
 mp_enableroundwaittime 1
 mp_restartround 0
 mp_bonusroundtime 19
-mp_chattime 20
+mp_chattime 10
```

Comment:
`mp_chattime` is as high as it is right now to allow possible runoff mapvotes to pass before the mapchange occurs. If we lower this we'd need to tweak other things e.g. voting time, bonus round time, whether runoffs can occur etc... which I think as it is now is OK.

Reply:
Ah, race conditions at its finest, lol. That's okay though, maybe I'll have a look into the mapchooser and submit a different pr for this.